src/services/cacheService.ts

```diff
@@ -429,6 +430,11 @@ export class CacheService {
       .then((response) => {
         if (response.ok) {
           return response.text().then((value) => {
+            let valueContentType = ContentType.StringContentType;
+            if (isNaN(Number(value)) && RestUtils.isJSONObject(value)) {
```

Comment:
no

Reply:
I have issues with that, I'm planning with gustavo some improvements in getEntry and getEntries because we know the Key Content Type but we don't know the Value Content type. We use the general content type header, but the content type header references the content type of the response coming, but this is not the content type of an entry. so we need to add a new header for Value-Content-Type to deal all those cases well.
12.1.x or 13

Reply:
I'd expect that calling ```getEntry``` with accept ```application/json``` would return everything in JSON format, not sure about adding more headers.

What is the issue, the JIRA does not describe from a REST API point of view

Reply:
Could you open a JIRA with the REST calls (+ headers) used?

Reply:
@gustavonalle which jira do you mean?

Reply:
This PR has a title "Entry edition not working due to content type", but it does not tell why, so I'd like to have the REST calls (with headers) that the console is using to reproduce the issue and maybe fix without having to add more headers or change the API

Reply:
the fix is done for proto caches. but for non proto caches and not json, and not string, not completely

@danberindei the changes of package are due to pass prettier, so I rather commit them